lib/include/event-driven/vDrawSkin.h

```diff
@@ -0,0 +1,1159 @@
+
+    public:
+
+        std::map<int,std::tuple<double, double>> pos {};
+
+        int xoffset,yoffset,scaling,radius,noise,max_value;
+
+
+        loadMap(){
+
+            noise = 2500;
+            max_value = 15000;
+
+            xoffset = 80;
+            //yoffset = 120;
+            yoffset = 680;
+            scaling=20;
+            radius =10;
+
+        };
+
+        void initialise(std::string bodypart){
+
+            std::vector<int> taxel2Repr;
+            yarp::os::ResourceFinder rf1, rf2;
+            std :: string filename1,filename2;
+
+            int check=0;
+
+            if (bodypart == "arm"){
+                Triangle_10pad skin;
+
+                filename1 = "skinGui/right_forearm_V2.ini";
+                filename2 = "positions/right_forearm_V2.txt";
+
+                rf2.setVerbose(true);
+                rf2.setDefaultContext("skinGui");            //overridden by --context parameter
+                rf2.setDefaultConfigFile(filename2); //overridden by --from parameter
+                rf2.configure(0,NULL);
+
+
+                if (rf2.check("taxel2Repr")){
+                    check+=1;
+                    std :: cout << "Calibration data";
+                    yarp::os::Bottle calibration_data = *(rf2.find("taxel2Repr").asList());
+                    for (int i = 0; i < calibration_data.size(); i++){
+                            taxel2Repr.push_back(calibration_data.get(i).asInt());
+                    }
+                    skin.initRepresentativeTaxels(taxel2Repr);
+                }
+
+                rf1.setVerbose(true);
+                rf1.setDefaultContext("skinGui");            //overridden by --context parameter
+                rf1.setDefaultConfigFile(filename1); //overridden by --from parameter
+                rf1.configure(0,NULL);
+
+
+                if (rf1.check("SENSORS")){
+                    check+=1;
+                    std:: cout<< "\n Sensors data"<<std::endl;
+                    yarp::os::Bottle &ini_data = rf1.findGroup("SENSORS");
+
+                    for (int i = 1; i < (ini_data.size()); i++){
+
+                        yarp::os::Bottle sensorConfig(ini_data.get(i).toString());
+
+                        //std::string type(sensorConfig.get(0).asString()); useless since forearm has always same type and hand has a precise numbering for palm and fingers.
+
+                        int id=sensorConfig.get(1).asInt();
+                        double x=sensorConfig.get(2).asDouble();
+                        double y=sensorConfig.get(3).asDouble();
+                        double orientation=sensorConfig.get(4).asDouble();
+                        double gain=sensorConfig.get(5).asDouble();
+                        int    mirror=sensorConfig.get(6).asInt();
+                        int    layoutNum=sensorConfig.get(7).asInt();
+
+                        skin.get_data(id,x,y,orientation,gain,mirror,layoutNum);
```

Comment:
`layoutNum` is not used.
```suggestion
                        skin.get_data(id,x,y,orientation,gain,mirror);
```

Reply:
agreed - at this point I'm not sure if it will break something else so I'll leave it as is.